backend/src-common/src/main/java/org/eclipse/sw360/components/summary/ComponentSummary.java

```diff
@@ -103,6 +103,9 @@ private Component makeExportSummary(Component document, List<Release> releases)
         copyField(document, copy, Component._Fields.CREATED_BY);
         copyField(document, copy, Component._Fields.CREATED_ON);
         copyField(document, copy, Component._Fields.VENDOR_NAMES);
+        copyField(document, copy, Component._Fields.MAIN_LICENSE_IDS);
```

Comment:
This may effect the spreadsheet export as well

Reply:
I checked this issue, but It did not effect the spreadsheet export function